src/Components/ArtCard/ArtCard.tsx

```diff
@@ -1,8 +1,25 @@
-import React from 'react'
+import React, { useState } from 'react';
+import './ArtCard.css';
+import FaveButton from '../FaveButton/FaveButton';
+import { ArtData }from '../../utils'
+
+
+interface Props {
+  pieces: ArtData[],
+  setPieces: (value: React.SetStateAction<ArtData[]>) => void,
+  setSavedPieces: (id: number | string) => void, 
+  piece: ArtData
+}
+
+const ArtCard = ({pieces, setPieces, setSavedPieces, piece}: Props) => {
+
+  const [artPiece, setArtPiece ] = useState(false)
```

Comment:
What is this artPiece boolean tracking exactly?   I wonder if changing the naming could make that more clear?